parallel_gripper_controller/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.16)
-project(parallel_gripper_controller LANGUAGES CXX)
+project(parallel_gripper_controller)
```

Comment:
Here it is missing

```
# using this instead of visibility macros
# S1 from https://github.com/ros-controls/ros2_controllers/issues/1053
set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Reply:
There's also a comment a line below this that we don't support these controllers on windows or mac

Reply:
Ahh crap. I totally missed it. Thanks